app/mailers/notifier.rb

```diff
@@ -1,27 +1,9 @@
 class Notifier < ApplicationMailer
 
-  def lost_password(user, password)
-    # Email header info MUST be added here
-    recipients user.email
-    from "#{$SUPPORT_EMAIL}"
-    subject "[#{$ORG_SITE}] Password Reset"
-
-    # Email body substitutions go here
-    body :user => user, :password => password
-  end
-
   def error(error)
-    recipients $ERROR_EMAIL
-    from "#{$ADMIN_EMAIL}"
-    subject "Exception Mailer"
-    body :exception_message => error.message, :trace => error.backtrace
-  end
-
-  def endlessloop(node)
-    recipients $ERROR_EMAIL
-    from "#{$ADMIN_EMAIL}"
-    subject "Exception Mailer"
-    body :node => node.inspect
+    mail(to: "#{$SUPPORT_EMAIL}", from: "#{$SUPPORT_EMAIL}",
+        subject: "[#{$SITE}] Exception Mailer: #{error.message}",
+        body: "#{error.backtrace}")
```

Comment:
Do as the feedback, add a view with a better display of the backtrace.
Displaying only the `#{error.backtrace}`is not readable.

Reply:
done